### DISCUSSION TITLE
PWA Release

### DISCUSSION THREAD

**Discussion by 4gray** [4gray/iptvnator#112] [Show and tell]
### ✋ Hello

Today I am happy to announce that IPTVnator is now also available as a progressive web application (PWA), which means that you can open the player directly from your browser. Thus the application became available, for example, for users of smartphones or tablets. At the moment, it is available at:

[https://iptvnator.vercel.app/](https://iptvnator.vercel.app/)

![IMG_20220109_142900_941](https://user-images.githubusercontent.com/1503032/148686260-286b5d3b-6b75-44a9-bc2a-abb088b1c8a7.jpg)

### 🚀 Features 

At the moment, the PWA version of the application is a light version and does not include some features which are available in the electron version. For example, the user agent specification is not available due to browser restrictions. Also, the program guide (EPG) function is not available yet, as it requires some reworking.

### 🤖 Versioning and further development:

In the future, both versions of the application will be released in parallel, since they are based on the same code base. The electron version of the application will be released about once every couple of months, as before, depending on development activity. The PWA version will be updated regularly, namely with each new commit.

<hr /><em>This discussion was created from the release <a href='https://github.com/4gray/iptvnator/releases/tag/pwa-1'>PWA Release</a>.</em>

**Comment by tvbsa** [comment]
Hello

How can I contact you?  Whatsapp me if possible please +2761 456 6007  Paul Smith

**Comment by jind75** [comment]
the media could not be loaded, either because the server or network faild or because format is not supported.

NOT PLAYING

**Comment by CrowVip** [comment]
I try to release the m3u link and get this message
"The media could not be loaded, either because the server or network failed or because the format is not supported."

Can you fix it?

**Reply by CrowVip** [reply]
Can you fix it?
